healtcheck.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"github.com/juju/errgo"
+)
+
+const (
+	StatusHealthy   = "healthy"
+	StatusUnhealthy = "unhealthy"
+)
+
+type HealthInfo struct {
+	Status   string       `json:"status"`
+	App      string       `json:"app"`
+	Version  string       `json:"version"`
```

Comment:
Required by what? I think `Status` and `App` should be required to be able to track a state chain properly.